index.js

```diff
@@ -66,7 +66,42 @@ function toggleColor(element) {
         element.style.color = "green"
     }
 }
+//added event listener
+header.addEventListener('click', function(event) {
+    
+   toggleColor(event.target)
+})
```

Comment:
great work! We have the header variable already defined earlier in the file.
All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered.

Great work incorporating the `toggleColor` function. We just need to pass in the event target as an argument to the function.

```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})

```

With ES6 notation

```
header.addEventListener("click", e => toggleColor(e.target))
```